data-import/src/main/resources/folijet/data-import/features/data-import-set-for-deletion.feature

```diff
@@ -253,7 +253,7 @@ Feature: Set for deletion logic
     # Verify instance mark as deleted
     And assert createdInstance.staffSuppress == true
     And assert createdInstance.discoverySuppress == true
-    And assert createdInstance.deleted == false
+    And assert createdInstance.deleted == true
```

Comment:
we must not change this verification because the test checks the unmarking of instance as deleted. Looks like it is just a typo in the comment above